packages/vite-environment-provider-cloudflare/src/moduleFallback.ts

```diff
@@ -15,101 +15,86 @@ export type ResolveIdFunction = (
 ) => Promise<string>;
 
 export function getModuleFallbackCallback(resolveId: ResolveIdFunction) {
-  return async (request: Request): Promise<Response> => {
-    const { resolveMethod, referrer, specifier, rawSpecifier } =
-      extractModuleFallbackValues(request);
+  return patchModuleFallbackHandler(
+    async (request: Request): Promise<Response> => {
+      const { resolveMethod, referrer, specifier, rawSpecifier } =
+        extractModuleFallbackValues(request);
 
-    const referrerDir = dirname(referrer);
+      let fixedSpecifier = specifier;
 
-    let fixedSpecifier = specifier;
+      fixedSpecifier = rawSpecifier;
```

Comment:
Why is `fixedSpecifier` assigned and then immediately reassigned?

Reply:
oops! 😓 

no, I meant to completely remove `fixedSpecifier`! thanks for noticing this! 🙏